src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Stylesheet with an SCSS extension won't be directly usable in a browser. You should link to a CSS file that is the compiled result of your SCSS file.

Reply:
it still compiles it in css bundle. sorry, but your current point is not relevant